<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mini Photoshop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QFrame" name="toolPanel">
      <property name="minimumWidth">
       <number>140</number>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="toolsLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tools</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="placeholderToolButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Coming soon</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="toolSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="canvasFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="canvasLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="canvasLabel">
         <property name="text">
          <string>Canvas</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyGraphicsView" name="graphicsView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="filterPanel">
      <property name="minimumWidth">
       <number>260</number>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="filterLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="filtersLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Filter parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bwButton">
         <property name="text">
          <string>Black &amp; White</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="filterScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>216</width>
            <height>1028</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="zoomLabel">
             <property name="text">
              <string>Zoom</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="scaleSlider">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="tracking">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="brightnessLabel">
             <property name="text">
              <string>Brightness</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSlider" name="brightnessSlider">
             <property name="minimum">
              <number>-255</number>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="contrastLabel">
             <property name="text">
              <string>Contrast</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSlider" name="contrastSlider">
             <property name="minimum">
              <number>-255</number>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="saturationLabel">
             <property name="text">
              <string>Saturation</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSlider" name="saturationSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="temperatureLabel">
             <property name="text">
              <string>Temperature</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSlider" name="temperatureSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="tintLabel">
             <property name="text">
              <string>Tint</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSlider" name="tintSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="exposureLabel">
             <property name="text">
              <string>Exposure</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSlider" name="exposureSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="gammaLabel">
             <property name="text">
              <string>Gamma</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSlider" name="gammaSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="vibranceLabel">
             <property name="text">
              <string>Vibrance</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QSlider" name="vibranceSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="shadowLabel">
             <property name="text">
              <string>Shadows</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSlider" name="shadowSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="highlightLabel">
             <property name="text">
              <string>Highlights</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QSlider" name="highlightSlider">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="clarityLabel">
             <property name="text">
              <string>Clarity</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QSlider" name="claritySlider">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="blurLabel">
             <property name="text">
              <string>Blur</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QSlider" name="blurSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="sharpLabel">
             <property name="text">
              <string>Sharpness</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QSlider" name="sharpSlider">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="vignetteLabel">
             <property name="text">
              <string>Vignette</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QSlider" name="vignetteSlider">
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <widget class="QLabel" name="grainLabel">
             <property name="text">
              <string>Grain</string>
             </property>
            </widget>
           </item>
           <item row="15" column="1">
            <widget class="QSlider" name="grainSlider">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="16" column="0">
            <widget class="QLabel" name="splitToningLabel">
             <property name="text">
              <string>Split toning</string>
             </property>
            </widget>
           </item>
           <item row="16" column="1">
            <widget class="QSlider" name="splitToningSlider">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="17" column="0">
            <widget class="QLabel" name="fadeLabel">
             <property name="text">
              <string>Fade</string>
             </property>
            </widget>
           </item>
           <item row="17" column="1">
            <widget class="QSlider" name="fadeSlider">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuRotate">
     <property name="title">
      <string>Rotate</string>
     </property>
     <addaction name="actionRotateLeft"/>
     <addaction name="actionRotateRight"/>
    </widget>
    <widget class="QMenu" name="menuFlip">
     <property name="title">
      <string>Flip</string>
     </property>
     <addaction name="actionFlipHorizontally"/>
     <addaction name="actionFlipVertically"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCrop"/>
    <addaction name="menuRotate"/>
    <addaction name="menuFlip"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
    <addaction name="actionResetFilters"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionResetZoom"/>
    <addaction name="actionFitToScreen"/>
    <addaction name="actionPan"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFilters"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionFlipHorizontally">
   <property name="text">
    <string>Horizontally</string>
   </property>
  </action>
  <action name="actionFlipVertically">
   <property name="text">
    <string>Vertically</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionResetZoom">
   <property name="text">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actionFitToScreen">
   <property name="text">
    <string>Fit To Screen</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="actionResetFilters">
   <property name="text">
    <string>Reset Adjustments</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>MyGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
